### DISCUSSION TITLE
新手小白挂载115打不开视频啊

### DISCUSSION THREAD

**Discussion by comtezhang** [AlistGo/alist#7445] [Q&A]
昨晚上刚接触alist，115网盘也挂上了，kodi也打开了，但马上就不能用了，今天早上也是，打开视频就提示重新连接1 2 3 4。或者弹出一大篇文字说触发了什么，禁止访问了。

**Comment by snugduahn** [comment]
触发了安全被阻断？刚登录的时候是可以用的，但没过几分钟就出现这种情况？

**Reply by comtezhang** [reply]
对啊对啊

很抱歉，由于您访问的URL有可能对网站造成安全威胁，您的访问被阻断。

**Reply by blss65** [reply]
在储存中设置115的分页大小，把分页调到最大，直接输两万继续，然后保持就好了。
